### PR TITLE
main/nettle: upgrade to 3.5.1

### PR DESCRIPTION
**Pull request by J0WI**
breaks ABI:
https://lists.lysator.liu.se/pipermail/nettle-bugs/2019/007564.html

### CODE REVIEW

**Comment by J0WI** [comment]
Added `--disable-static` from #9175

**Comment by J0WI** [comment]
@fabaff could you review this?

File: main/nettle/APKBUILD [hunk 1]
```diff
@@ -2,28 +2,19 @@
...
 
 build() {
-	local _arch_opts=
-
-	case "$CARCH" in
-	armhf|armv7)
-		_arch_opts="--disable-arm-neon"
-	esac
-
 	cd "$builddir"
```
Comment (fabaff) [inline main/nettle/APKBUILD]: This is no longer needed.

Comment (J0WI) [reply main/nettle/APKBUILD]: Done

**Comment by J0WI** [comment]
ea8095d2f0259e83209946a44dbac891800db189

**Pull request closed without merging**